examples/README.md

```diff
@@ -75,6 +75,8 @@ your Ceph cluster setup.
 If you followed the documentation to create the rbdplugin, you shouldn't
 have to edit any other file.
 
+Note that it is recommended to snapshot only when the volume is not in use.
```

Comment:
same here:
` Note that it is recommended to create a snapshot only when the PV is not in use.`

Reply:
Done